pkg/controller/manager/manager_controller.go

```diff
@@ -166,11 +166,6 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	if err = utils.AddNamespaceWatch(managerController, common.TigeraPrometheusNamespace); err != nil {
 		return fmt.Errorf("manager-controller failed to watch the '%s' namespace: %w", common.TigeraPrometheusNamespace, err)
 	}
-
-	if err = utils.AddConfigMapWatch(managerController, render.ECKLicenseConfigMapName, render.ECKOperatorNamespace, eventHandler); err != nil {
```

Comment:
:thinking: Why were we watching the license here in the first place ?

Reply:
No clue, that's why I removed it :sweat_smile: